release_docs/RELEASE.txt

```diff
@@ -47,6 +47,12 @@ New Features
 
     Configuration:
     -------------
+    - Added new configure option to support building parallel tools.
+      See Tools below (autotools - CMake):
+            --with-libmfu=DIR      HDF5_BUILD_PARALLEL_TOOLS
```

Comment:
Yeah... The CMake version indirectly uses FindMFU.cmake, etc. to hopefully locate the necessary libraries and include directories so that dh5walk can be built.   I'm happy to make these more similar, do you have a suggestion?